static/js/l10n/i18n.tsx

```diff
@@ -0,0 +1,35 @@
+import { createIntl, createIntlCache } from "react-intl";
```

Comment:
nice

since this file doesn't include JSX, the file extension should be .ts

Reply:
done